community/kernel/src/main/java/org/neo4j/kernel/api/index/ArrayEncoder.java

```diff
@@ -39,11 +38,21 @@ protected void encodeBufferPrefix( OutputStream out ) throws IOException
         }
 
         @Override
+        protected void encodeBufferSuffix( OutputStream outputStream ) throws IOException
+        {
+            // nothing to do here
```

Comment:
I still would have liked these comments to state _why_ these methods are overridden, rather than just a "this method intentionally left blank".

Something like "suppress the behaviour of the parent class and ensure that no extra characters are emitted at the end of the buffer".